2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -343,6 +342,74 @@ class CollectorService(
     if (pixelExpected) List(`Cache-Control`(`no-cache`, `no-store`, `must-revalidate`))
     else Nil
 
+  /**
+   * Determines the cookie domain to be used by inspecting the Origin header of the request
+   * and trying to find a match in the list of domains specified in the config file.
+   * The Origin header may include multiple domains. The first matching domain is used.
+   * If no match is found, the cookie domain is not set.
+   */
```

Comment:
It'd be great to have by-parameter scaladoc here I think

Reply:
Done.